charts/unauthenticated-ingress/values.yaml

```diff
@@ -0,0 +1,33 @@
+# Default values for standard-application.
+
+#
+# ReplicaCount denotes how many pods you want
+#
+replicaCount: 1
+
+nameOverride: ""
+fullnameOverride: ""
+
```

Comment:
remove these unused values

Reply:
Removed other unused values but these are used in templates/_helpers.tpl